### DISCUSSION TITLE
Translation Language Requests

### DISCUSSION THREAD

**Discussion by arsenetar** [hsoft/dupeguru#997] [Contribution]
Since Transifex does not really support a good method to discuss new language requests, this discussion is for making those requests.

To see existing languages and languages that can be requested see: https://www.transifex.com/voltaicideas/dupeguru-1.

If requesting a new language to be added please post here with:
1. The language being requested including the language code (ex. English en-US).
2. The rational for adding the language. For a language that does not already have a variant in the project or does not have a variant then nothing additional is needed.  For instances where a variant already exists additional information is needed:
  - If another variant exists (ex. en-US and en-GB is requested) then rational for the difference should be provided, as supporting all variants may not make sense.
  - If a "base" variant exists (ex. en and en-GB is requested) then there needs to be some discussion on if the "base" version should be moved to a variant (ex. en-US lets say in this example) or if the "base" is actually currently the same as the variant and should move to it for clarity.    In short how should the clarity between the existing "base" and the variant be maintained. 

The additional information is helpful when deciding how to manage the languages added to the project as not everyone knows the various nuances between the different variants of the languages or what state the translations are currently in.

**Comment by comradekingu** [comment]
Happy to help add nb_NO if you move to Weblate.
Right now there is an empty file for Norwegian Bokmål, which looks to be "nb", which isn't correct.
Deleting this would be preferable.

**Reply by arsenetar** [reply]
Not really looking to move to a different translation platform at this time.  Not sure what you mean by the Norwegian Bokmål (nb) file not being correct, it has no translations at this time.  Translations are not added to the application until they are mostly complete (>~70%).
